packages/create-cloudflare/src/helpers/latestPackageVersion.ts

```diff
@@ -0,0 +1,14 @@
+import { fetch } from "undici";
+
+interface NpmInfoResponse {
+	"dist-tags": { latest: string };
+}
+
+/**
+ * Get the latest version of an npm package by making a request to the npm REST API.
+ */
+export async function getLatestPackageVersion(packageSpecifier: string) {
```

Comment:
Not a huge deal, but I think that most if not all functions in create-cloudflare are defined as arrow functions

I personally do prefer standard clear `function`s but maybe we should try to be consistent in this?

Reply:
never mind... I can see that we actually have a bit of a mix of both 😅